### ISSUE TITLE
Hovered button text not readable in Dark + Monochrome mode

### ISSUE THREAD

**Issue by Qianqianye** [processing/p5.js-website#496]
### Most appropriate sections of the p5.js website?

Rounded Button component across the website.

### Actual Behavior

When Dark and Monochrome modes are both toggled on from the 'Accessibility' drop down menu, the text in the hovered buttons, like 'Start Coding', 'Donate', and 'Access Statement', are not readable because they have the same color as the background, as shown at screenshot below.

This is related to issue #468. The solution might be similar to PR #494 

### Hovering on 'Donate' button at left column:
<img width="845" alt="image" src="https://github.com/user-attachments/assets/84adca0d-310d-4968-929a-2d424ca4205c">

### Hovering on 'Access Statement' button on 'About' page:
<img width="781" alt="image" src="https://github.com/user-attachments/assets/35316bcf-8f09-4d63-8990-94917a323bf9">

### Expected Behavior

The button text should be black in the button hover state.

### Would you like to work on the issue?

Would love to assign someone else

**Comment by mayaarguelles** [comment]
Hi, I'd be happy to help out if you'll assign the issue to me!

**Comment by Qianqianye** [comment]
Thanks @mayaarguelles. Just assigned you. Please feel free to let us know if you have any questions!
